### PR TITLE
[DC-2064] [Circle CI] Fix NotFound error in delete_stale_test_datasets

### PR DESCRIPTION
**Pull request by hiro-mishima**
Please see the JIRA ticket for the details.

### CODE REVIEW

File: data_steward/tools/delete_stale_test_datasets.py [hunk 1]
```diff
@@ -59,19 +59,27 @@ def _filter_stale_datasets(bq_client, first_n: int = None):
...
+            LOGGER.info(
+                f"{n}: stale_dataset={dataset_name}, time_created={dataset_created}."
+            )
+            n += 1
+
+        except exceptions.NotFound as e:
+            LOGGER.info(
+                f"{dataset_name} not found. This dataset is likely to be"
+                f"a temporary dataset that was created and deleted by other job or test."
+                f"Message: {e.message}")
```
Comment (nishanthpp93) [inline data_steward/tools/delete_stale_test_datasets.py:82]: nit: The string gets stitched together while logging, so I'd suggest adding spaces after each line
```suggestion
                f"{dataset_name} not found. This dataset is likely to be "
                f"a temporary dataset that was created and deleted by other job or test. "
                f"Message: {e.message}")
```

**Review by nishanthpp93 — COMMENTED** [review]
Looks good! Thanks for quickly addressing this. Added a small nit.

File: data_steward/tools/delete_stale_test_datasets.py (hunk 1 above)
Comment (hiro-mishima) [reply data_steward/tools/delete_stale_test_datasets.py]: Thanks, @nishanthpp93 ! That was a rookie mistake... I fixed it.

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Pull request merged**